src/manifest.json

```diff
@@ -30,7 +30,8 @@
     "storage",
     "tabs",
     "webRequestBlocking",
-    "webRequest"
+    "webRequest",
+    "theme"
```

Comment:
I'm asking our @mozilla/addons-editorial team if this will trigger a new permission prompt to users. That's usually a UX bump that we like to avoid if we can.

Reply:
Answer: no, [adding `theme` will not re-prompt the user for permission](https://searchfox.org/mozilla-central/source/toolkit/components/extensions/test/xpcshell/test_ext_permissions.js#562-585).